src/js/component/panel/PanelIssues.js

```diff
@@ -119,25 +119,45 @@ function getOrderedFilteredIssues({
         });
     }
 
-    // Sort by priority, then owner
-    preparedIssues = _.sortBy(preparedIssues, (item) => {
-        const priority = priorities[item.url ?? ''] && (priorities[item.url].priority !== undefined)
+    // Sort by priority, then owner. Sorting has to be chained, since _.sortBy doesn't support multi-criteria sorting.
+    // If an array is returned, it will be sorted lexicographically and cause issues above 10 items.
+    // Iteratee for sorting by owner (secondary sort, for unprioritized items)
+    const ownerSortIteratee = (item) => {
+        const priorityValue = priorities[item.url ?? ''] && (priorities[item.url].priority !== undefined)
             ? priorities[item.url].priority
             : Number.MAX_SAFE_INTEGER;
-        return [priority, item.currentUserIsOwner ? 0 : 1];
-    });
+        if (priorityValue === Number.MAX_SAFE_INTEGER) {
+            return item.currentUserIsOwner ? 0 : 1; // Sort unprioritized by owner
+        }
+        return 0; // Prioritized items get the same key here to maintain stability for the next sort
+    };
+
+    // Iteratee for sorting by priority (primary sort)
+    const priorityIteratee = (item) => {
+        // If an issue doesn't have a priority, return -1 so that is appears at the top of the list, which will prompt the user to set a priority
+        const priorityValue = priorities[item.url ?? ''] && (priorities[item.url].priority !== undefined)
+            ? priorities[item.url].priority
+            : -1;
+
+        return priorityValue;
+    };
```

Comment:
```suggestion
const priorityIteratee = (item) => priorities[item.url ?? '']?.priority ?? -1;
```
NAB: this can be simplified

Reply:
same as above